examples/CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ function(assign_ir target binary_name input private_input curve_type max_num_pro
         COMMAND_EXPAND_LISTS
         VERBATIM)
 
+    add_custom_target(${target}_estimate_size
```

Comment:
How about some tests on this mode? Even simple runs of examples without checking results would be better then nothing.

Reply:
done